Updated CWS and GEOT given the new data

robfairh opened the pull request:
Fixes #150. Here is the figure of the new model:

![13_uiuc](https://user-images.githubusercontent.com/43791150/106954901-32d82e00-6713-11eb-9239-a1f89a7ebc67.png)

As ```GEOT``` has two outputs that have different units (MWth and tons) I preferred to split the conversion from ```ELC``` to ```UCWS``` into two steps. ```GEOT``` converts MWe into MWth. And ```DGEOT``` converts MWth into tons.
This way, we can calculate ```TechOutputSplit``` using the thermal outputs from ```GEOT```.
Let me know what you think about this @samgdotson.

samgdotson committed:
begin update to cws

robfairh committed:
scenario works

robfairh committed:
added geothermal assumptions in md

robfairh committed:
updated values from GEOT

robfairh committed:
13_uiuc separates USTM and CHWS

robfairh committed:
updated assumptions

samgdotson commented on temoa-uiuc/data_files/13_uiuc.sql:
```diff
@@ -0,0 +1,1743 @@
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'GEOT', 2024, 'USTM', 3.74, 'converts electricity to thermal energy');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'GEOT', 2025, 'USTM', 3.74, 'converts electricity to thermal energy');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'GEOT', 2026, 'USTM', 3.74, 'converts electricity to thermal energy');
+INSERT INTO "Efficiency" VALUES('uiuc', 'ELC', 'GEOT', 2027, 'USTM', 3.74, 'converts electricity to thermal energy');
```
These units, with the DCW commodity, give a COP that is greater than 5. Is this intentional?

I would do the following:
* what fraction of the output goes to chilled water?
* what fraction of the output goes to steam?
Then use the COP for the efficiencies in the intermediate technologies. 
E.g. 
60% of the geothermal energy goes to steam, so we have a GSTM commodity, and GSTM --> UH with efficiency=COP. Does that make sense?
Rather than DCW --> DGEOT, have DCW --> CHWS with efficiency COP*conversion to tons

robfairh replied:
For the heating part:
ELC  [MWe] -> x COP = 3.74 -> USTM [MWth]

For the cooling part:
ELC  [MWe] -> x COP = 4.28 -> DCW [MWth] -> 285 (unit conversion) -> UCWS [ton-hours]

samgdotson reviewed (changes requested):
It's okay, check the efficiencies, get rid of DGEOT and have CHWS efficiency = COP for DCW--> CHWS. See comment

robfairh commented:
I think I understood. That's what I am trying to avoid. Cause I need the capacity of the geothermal plant. If the outputs have different units, how do we calculate the capacity? My way, the outputs have the same units, and obtaining a capacity is easier.

robfairh committed:
added clarification

robfairh committed:
added chilled water to data parser

robfairh commented:
Here are some figures with the results:

* CHW CAPACITY
![scenario13_chw_capacity](https://user-images.githubusercontent.com/43791150/107803047-dbecdd00-6d40-11eb-8602-ecf5df00b805.png)

* CHW GENERATION
![scenario13_chw_generation](https://user-images.githubusercontent.com/43791150/107803050-ddb6a080-6d40-11eb-9e74-60c9b2579d6a.png)

* IND CAPACITY
![scenario13_ind_capacity](https://user-images.githubusercontent.com/43791150/107803057-dee7cd80-6d40-11eb-8128-025d918638d5.png)

* IND GENERATION
![scenario13_ind_generation](https://user-images.githubusercontent.com/43791150/107803060-e018fa80-6d40-11eb-95f6-55128881e97f.png)

robfairh commented:
@samgdotson it seems like the capacity of the geothermal plant is counted both for the steam and the chilled water production.
This is the problem with having both plants as only one technology.
We might need to separate the technologies after all.
The problem is that TEMOA will most likely deploy steam and chilled water geothermal plants with capacities that are not correlated to each other, which is not true in real life.

robfairh committed:
changes in assumptions

robfairh committed:
updated assumptions: added existing cap

robfairh committed:
more changes in assumptions and sql

samgdotson committed:
updates assumptions for geot. need to make CHWS in tons and ton hours

samgdotson committed:
fixes the units of chws

samgdotson committed:
fixes units of existing capacity for chilled water

samgdotson committed:
fixes costs

robfairh committed:
changes in assumptions for geot and chws

robfairh committed:
cleaned assumptions

robfairh committed:
fixed assumption in cws

robfairh committed:
fixed units of cooling: kilo-tons

robfairh commented:
@samgdotson this is ready for review. I cleaned the jupyter notebooks and I fixed a few things. Among those things, we were confusing mega-tons with kilo-tons in some tables. Similarly to GWh and MW, the activity units for UCWS should be mega-ton-hours, while for the capacity, the units should be kilo-tons. I corrected some numbers based on that.

samgdotson reviewed (approved):
The changes look good to me! @yardasol will need to adopt these changes to the extended model.

samgdotson commented:
Merging, inspite of failed checks -- circle ci config file is not properly set up.
